ArcadiaParties-SPA/src/app/services/unauthorized-interceptor.ts

```diff
@@ -1,35 +0,0 @@
-import { Injectable } from '@angular/core';
```

Comment:
Why did you remove it?

Reply:
We can catch errors in the same interceptor, where we add header

Reply:
We can catch 401 error in the same interceptor, where we add header